### PR TITLE
[DOCS-3673] Add Log Forwarding doc

### PR DESCRIPTION
**Pull request by maycmlee**
### What does this PR do?

Adds new page and images for Log Forwarding.

### Motivation

New feature.

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/logs/log_configuration/forwarding_custom_destinations.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+{{< beta-callout url="#" btn_hidden="true">}}
+  Log Forwarding is currently in beta.
+{{< /beta-callout >}} 
+
+## Overview
+
+Log Forwarding allows you to send logs from Datadog to custom destinations like Splunk, Elasticsearch, and HTTP endpoints. This means that you can use [Log Pipelines][1] to centrally collect, process, and standardize your logs in Datadog. Then, send the logs from Datadog to other tools to support individual teams’ workflows. You can choose to forward any of the ingested logs, whether or not they are indexed, to custom destinations. Logs are forwarded in JSON format and compressed with GZIP.
+
+{{< img src="logs/log_configuration/forwarding/forwarding_page.png" alt="The log forwarding page showing custom destinations highlighted and splunk, HTTP endpoint, and Elasticsearch in the list of destinations" >}}
```
Comment (cswatt) [inline content/en/logs/log_configuration/forwarding_custom_destinations.md:23]:
```suggestion
{{< img src="logs/log_configuration/forwarding/forwarding_page.png" alt="The Log Forwarding page, showing custom destinations highlighted. The list of destinations includes Splunk (filtered by service:logs-processing), HTTP Endpoint (filtered by source:okta OR source:paloalto), and Elasticsearch (filtered by team:acme env:prod)." >}}
```

File: content/en/logs/log_configuration/forwarding_custom_destinations.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+**Note:** Only Datadog users with [logs_write_forwarding_rules][2] permission can create, edit, or delete custom destinations for forwarding logs.
```
Comment (cswatt) [inline content/en/logs/log_configuration/forwarding_custom_destinations.md:25]:
```suggestion
**Note**: Only Datadog users with the [`logs_write_forwarding_rules`][2] permission can create, edit, or delete custom destinations for forwarding logs.
```

File: content/en/logs/log_configuration/forwarding_custom_destinations.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+## Set up log forwarding to custom destinations
+
+1. Navigate to [Log Forwarding][3]. Alternatively, go to **Logs** > **Configuration** and click the **Log Forwarding** tab.
+2. Select **Custom Destinations**.
+3. Click **New Destination**.
+4. Enter the query to filter your logs for forwarding. See [Search Syntax][4] for more information.
+5. Select the **Destination Type**.
+
+{{< img src="logs/log_configuration/forwarding/configuration.png" alt="The destination configuration page showing the steps to set up a new designation" style="width:70%;">}}
```
Comment (cswatt) [inline content/en/logs/log_configuration/forwarding_custom_destinations.md:35]:
```suggestion
{{< img src="logs/log_configuration/forwarding/configuration.png" alt="The destination configuration page, showing the steps to set up a new destination." style="width:70%;">}}
```

File: content/en/logs/log_configuration/forwarding_custom_destinations.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+8. In the **Configure Authentication** section, select one of the following authentication types and provide the relevant details:
+    - Basic Authentication: Provide the username and password.
+    - Request Header: Provide the header name and value. 
+9. Click **Save**.
+
+{{% /tab %}}
+
+{{% tab "Splunk" %}}
+
+6. Enter a name for the destination. .
```
Comment (cswatt) [inline content/en/logs/log_configuration/forwarding_custom_destinations.md:51]:
```suggestion
6. Enter a name for the destination.
```

File: content/en/logs/log_configuration/forwarding_custom_destinations.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+8. In the **Configure Authentication** section, enter the Splunk HEC token. 
+9. Click **Save**.
+
+{{% /tab %}}
+
+{{% tab "Elasticsearch" %}}
+
+6. Enter a name for the destination.
+7. In the **Configure Destination** section, enter the following details:
+    a. The endpoint to which you want to send the logs. The endpoint must start with `https://`.
```
Comment (cswatt) [inline content/en/logs/log_configuration/forwarding_custom_destinations.md:62]: there seems to be a Markdown issue with this sub-list

**Review by cswatt — CHANGES_REQUESTED** [review]
small grammar changes + there's a markdown indentation issue in step 7 of the Elasticsearch instructions, the nested list isn't rendering properly.

File: content/en/logs/log_configuration/forwarding_custom_destinations.md (hunk 5 above)
Comment (maycmlee) [reply content/en/logs/log_configuration/forwarding_custom_destinations.md]: Thanks, fixed!

**Review by cswatt — APPROVED** [review]

**Pull request merged**